dbt_style_guide.md

```diff
@@ -233,5 +355,37 @@ models:
 
 ## Jinja style guide
 
-* When using Jinja delimiters, use spaces on the inside of your delimiter, like `{{ this }}` instead of `{{this}}`
-* Use newlines to visually indicate logical blocks of Jinja
+- Jinja delimiters should have spaces inside of the delimiter between the brackets and your code.  
+  Example: `{{ this }}` instead of `{{this}}`
+- Use new lines to visually indicate logical blocks of Jinja or to enhance readability.  
+  Example:  
+  ```jinja 
+  {%- set orig_cols = adapter.get_columns_in_relation(ref('fct_orders')) %}
+
+  {%- set new_cols = dbt_utils.star(
+        from=ref('fct_order_items'),
+        except=orig_cols
+  ) %}
+
+  -- original columns
+  {%- for col in orig_cols %}
+  {{ col }}
+  {% endfor %}

```

Comment:
suggestion: inside for loops or if blocks, code should be indented
```suggestion
  -- original columns
  {%- for col in orig_cols %}
      {{ col }}
  {% endfor %}

```

Reply:
👍

Reply:
I feel like it makes sense in some cases, but in others, with lots of nested `for`/`if`, the compiled code then looks all over the place. If it makes my compiled code easier to read, I would go for 
```jinja
  {%- for col in orig_cols %}
  {{ col }}
  {% endfor %}
```
instead of the indented version.

Reply:
Last line on this style guide is 

>  Opt for code readability over compiled SQL readability.

So, I would disagree - the indenting makes the jinja code more readable

Reply:
Changing this to be indented for readability, but agree completely with Benoit. The rendered version gets really weird when you start indenting things all over the place. With that said though, we're going to leave it up to the reader on whether or not they want to indent this and try to adhere to what we're saying in the style guide.

Reply:
Let me know how this looks everyone! [Commit](https://github.com/dbt-labs/corp/pull/63/commits/313f683a97d9e513fa379bb43a89aa82e9b1cbdc)